test/pizza-guy.spec.js

```diff
@@ -50,14 +50,24 @@ describe('pizza-guy', function() {
 
   describe('onError', function() {
     it('should throw an error for non-function values', function() {
-      expect(() => pizzaGuy.onError(null)).toThrow('Must be a function');
+      expect(() => pizzaGuy.onError(null)).toThrow('The onError must be a function');
```

Comment:
I think that "the" is not required. Sounds better just "onError must be a function".